docs/csharp/tuples.md

```diff
@@ -16,7 +16,7 @@ ms.assetid: ee8bf7c3-aa3e-4c9e-a5c6-e05cc6138baa
 # C# Tuple types
 
 C# Tuples are types that you define using a lightweight syntax. The advantages
-include a simpler syntax, rules for conversions based on arity and types, and
+include a simpler syntax, rules for conversions based on arity and types of fields, and
```

Comment:
Instead of arity, I'd use number -- I'm not sure how many potential readers know what arity means.